src/js/components/settings/integrations.js

```diff
@@ -139,21 +143,51 @@ export const Integrations = ({ integrations = [], changeIntegration, createInteg
   };
 
   return (
-    <div>
-      <h2 className="margin-top-small">Integrations</h2>
-      {!isCreating && !!availableIntegrations.length && (
-        <Select displayEmpty onChange={onConfigureIntegration} value={currentValue} style={{ minWidth: 300 }}>
-          <MenuItem value="">Add new integration</MenuItem>
-          {availableIntegrations.map(item => (
-            <MenuItem key={item.provider} value={item.provider}>
-              {item.title}
-            </MenuItem>
+    <div style={{ minHeight: '95%', display: 'flex', flexDirection: 'column' }}>
+      <div>
+        <h2 className="margin-top-small">Integrations</h2>
+        {!isCreating && !!availableIntegrations.length && (
+          <Select displayEmpty onChange={onConfigureIntegration} value={currentValue} style={{ minWidth: 300 }}>
+            <MenuItem value="">Add new integration</MenuItem>
+            {availableIntegrations.map(item => (
+              <MenuItem key={item.provider} value={item.provider}>
+                {item.title}
+              </MenuItem>
+            ))}
+          </Select>
+        )}
+        {configuredIntegrations.map(integration => (
+          <IntegrationConfiguration key={integration.id} integration={integration} onCancel={onCancelClick} onDelete={deleteIntegration} onSave={onSaveClick} />
+        ))}
+      </div>
+      <div style={{ marginTop: 'auto' }}>
```

Comment:
If we want to add this for all the potential integrations in the future, the contents of this div would probably be better suited in the `<IntegrationConfiguration />` - but feel free to keep it as is, since we only have Azure so far...